### DISCUSSION TITLE
teamtalk5 and teamtalk5pro client in 5.9.2 beta

### DISCUSSION THREAD

**Discussion by Nardol** [BearWare/TeamTalk5#1354] [Q&A]
Hello,

The TeamTalk V5.9.2 beta for Ubuntu 18 have two binaries:
* teamtalk5
* teamtalk5pro

What are differences between these two client?
Is it expected? :slightly_smiling_face: usually only teamtalk5 is provided so I am curious.

**Comment by bear101** [comment]
Hm, yes, there shouldn't be a 'teamtalk5pro'. It will not work since libTeamTalk5Pro.so is missing. There's no difference between the two clients expect the name and the shared library they depend on.

**Reply by Nardol** [reply]
@bear101 FYI teamtalk5pro is still included with version 5.10.0.5087 for Ubuntu 18.
I don't know for other OS.

**Comment by lbk2907** [comment]
@Nardol For Windows, I not found any TeamTalk5Pro client anymore since the last version (5.10.0.5086).

**Reply by bear101** [reply]
Fixed: https://github.com/BearWare/TeamTalk5/commit/22dbc0137bb45c43dfc12e136cf2775a310fe785
